src/main/resources/templates/mdBenefitsFlow/mailingAddress.html

```diff
@@ -24,8 +24,7 @@
                 <th:block th:replace="~{fragments/inputs/address ::
                                   address(
                                     validate=true,
-                                    inputName='mailingAddress',
-                                    streetAddress2HelpText=#{general.optional}
+                                    inputName='mailingAddress'
                                   )}"/>
```

Comment:
I thought we talked about keeping the optional (at least as a smaller group).   Was there more conversation on this?